stemcell_builder/stages/logrotate_config/assets/opensuse-logrotate.conf

```diff
@@ -2,6 +2,10 @@
 # rotate log files weekly
 weekly
 
+#? use the syslog group by default, since this is the owning group
+#? of /var/log/syslog.
+# su root syslog
```

Comment:
just comments? And what's with the `?`?

Reply:
That's the funny thing... this file is actually the default that comes from logrotate, but i'll double check